src/server.js

```diff
@@ -51,6 +54,13 @@ module.exports = function() {
 		return res.sendFile(theme);
 	});
 
+	app.get("/packages/:package/:filename", (req, res) => {
+		const packageName = req.params.package;
+		const fileName = req.params.filename;
+		const packagePath = Helper.getPackageModulePath(packageName);
+		return res.sendFile(path.join(packagePath, fileName));
```

Comment:
I think we should also verify that `fileName` is in fact exposed by the package, e.g. the `stylesheets` array.

Reply:
Yeah, I guess there's a minor security concern if we don't check that. Added.